homeassistant/components/hue/bridge.py

```diff
@@ -201,6 +209,7 @@ async def hue_activate_scene(self, call, updated=False, hide_warnings=False):
         """Service to call directly into bridge to set scenes."""
         group_name = call.data[ATTR_GROUP_NAME]
         scene_name = call.data[ATTR_SCENE_NAME]
+        transition = call.data.get(ATTR_TRANSITION, DEFAULT_SCENE_TRANSITION)
```

Comment:
We don't need the default here since the service schema will set the default. Just use `call.data[ATTR_TRANSITION]`.

Reply:
I had thought that should be the case but was getting an error in my tests when I tried that stating that the key didn't exist in the dictionary

Reply:
We get an error since we don't use the service registry to call the service in the tests. The validation schema is applied in the service registry before the call comes into the integration.

This is an example why we should use the core interfaces to start the tests.